rfcs/0000-kuadrant-release-process.md

```diff
@@ -0,0 +1,207 @@
+|                                 | kuadrant-operator manifests                    | [OperatorHub.io](https://operatorhub.io/operator/kuadrant-operator)                    |
+
+## Documentation
+
+It's important to  note that keeping the documentation up to date is a responsibility of the component maintainers, and
+it needs to be done before releasing a new version of the component. The importance of keeping a clear and up-to-date
+documentation is crucial for the success of the project.
+
+The documentation for the Kuadrant suite is compiled and available on the [Kuadrant website](https://kuadrant.io/). One
+can find the source of the documentation within each component repository, in the `docs` directory. However, making this
+information available on the website is a manual process, and should be done by the maintainers of the project. The
+process of updating the documentation is simple and consists of the following steps:
+
+1. Update the documentation in the corresponding component repository.
+2. In [https://github.com/Kuadrant/docs.kuadrant.io/](https://github.com/Kuadrant/docs.kuadrant.io/), update the value of
+the `import_url` within the `multirepo` in the `plugins` section of the [mkdocs.yml](https://github.com/Kuadrant/docs.kuadrant.io/blob/main/mkdocs.yml)
+file, to point to the tag or branch of the component repository that contains the updated documentation.
+3. Once the changes are merged to main, the workflow that updates the website will be triggered, and the documentation
+will be updated.
+4. If for some reason it's needed to trigger the workflow manually, one can do it from the GitHub Actions tab in the
+[docs.kuadrant.io](https://github.com/Kuadrant/docs.kuadrant.io/actions/workflows/build.yaml) (`Actions > ci > Run Workflow`).
+
+## Communication
+
+Another important aspect of releasing a new version of the Kuadrant suite is the communication with the community and
+other teams within the organization. A few examples of the communication channels that need to be updated are:
+
+- Changelog generation
+- Release notes
+- Github Release publication
+- Slack channel in Kubernetes workspace
+- Blog post, if applicable
+- Social media, if applicable
+
+
+# Rationale and alternatives
+[rationale-and-alternatives]: #rationale-and-alternatives
+
+The alternative to the proposal is to keep the current process, which is not ideal and leads to confusion and lack of
+transparency.
+
+# Prior art
+[prior-art]: #prior-art
+
+There's been an organically grown process for releasing new versions of the Kuadrant suite, which is not documented and
+it's been changing over time. However, there are some documentation for some of the components, worth mentioning:
+
+* [Authorino release process](https://docs.google.com/document/d/1tLveyv8Zwe0wKyfUTWOlEnFeMB5aVGqIVDUjVYWax0U)
```

Comment:
I'm afraid this document is not accessible outside of Red Hat bu default. Shall we follow Limitador and move it into the Authorino repo?